### PR TITLE
Set up Theater mini app in Java Lab

### PR DESCRIPTION
**Pull request by jmkulwik**
This adds the TheaterVisualization and the Theater.js. It accepts a base64 encoded gif from Javabuilder and displays it on the canvas. 

Additionally, I did a little cleanup to remove DefaultVisualization and consolidate some logic for which visualization to display.

One other minor cleanup: We always want to show the speed slider in NeighborhoodVisualization and will always have ICON_PATH set to `/blockly/media/turtle/`. I cleaned these up too when the line length was flagged by our linter. It made more sense to put those values in NeighborhoodVisualization than to put them in Theater.js.

TheaterSample

## Links

- jira ticket: [CSA-429](https://codedotorg.atlassian.net/browse/CSA-429)

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/javalab/Javalab.js [hunk 1]
```diff
@@ -18,7 +18,8 @@ import {showLevelBuilderSaveButton} from '@cdo/apps/code-studio/header';
 import {RESIZE_VISUALIZATION_EVENT} from '@cdo/apps/lib/ui/VisualizationResizeBar';
 import Neighborhood from './Neighborhood';
 import NeighborhoodVisualizationColumn from './NeighborhoodVisualizationColumn';
-import DefaultVisualization from './DefaultVisualization';
```
Comment (molly-moen) [inline apps/src/javalab/Javalab.js:21]: should we delete the DefaultVisualization file now?

**Review by molly-moen — APPROVED** [review]
LGTM! Just one comment on potential clean-up

File: apps/src/javalab/Javalab.js (hunk 1 above)
Comment (jmkulwik) [reply apps/src/javalab/Javalab.js:21]: Already done :) git tracked it as a rename to TheaterVisualization (bit odd... but I guess the files were similar enough)

Comment (molly-moen) [reply apps/src/javalab/Javalab.js:21]: ah weird, I should have looked closer at the file names!

**Pull request merged**